integration/pom.xml

```diff
@@ -71,7 +71,7 @@
         <dependency>
             <groupId>com.aws.greengrass</groupId>
             <artifactId>client-devices-auth</artifactId>
-            <version>2.2.0-SNAPSHOT</version>
+            <version>2.4.0-SNAPSHOT</version>
```

Comment:
do not increase this. Changing it means we'd need to change our public requirements.